ddtrace/opentelemetry/span.go

```diff
@@ -70,8 +70,9 @@ func (s *span) extractTraceData(c *oteltrace.SpanContextConfig) {
 	}
 	state, err := oteltrace.ParseTraceState(headers["tracestate"])
 	if err != nil {
-		c.TraceState = state
+		return
 	}
+	c.TraceState = state
 	parent := strings.Trim(headers["traceparent"], " \t-")
```

Comment:
I think the other PR that was merged will fix this, so just need to finish the rebase it looks like.